File: Top Interview 150/88_Merge Sorted Array.py
# Python3

class Solution:
    def merge(self, nums1: List[int], m: int, nums2: List[int], n: int) -> None:
        """
        Do not return anything, modify nums1 in-place instead.
        """
        for i in range(m):
                nums1[i] = nums1[i]
        for j in range(n):
            nums1[m+j] = nums2[j]
        nums1 = nums1.sort()

# Python
class Solution(object):
    def merge(self, nums1, m, nums2, n):
        """
        :type nums1: List[int]
        :type m: int
        :type nums2: List[int]
        :type n: int
        :rtype: None Do not return anything, modify nums1 in-place instead.
        """
        for i in range(m):
                nums1[i] = nums1[i]
        for j in range(n):
            nums1[m+j] = nums2[j]
        nums1 = nums1.sort()

File: Top Interview 150/88_Merge Sorted Array.cpp
class Solution {
public:
    void merge(vector<int>& nums1, int m, vector<int>& nums2, int n) {
        //for i in range(m):
        //        nums1[i] = nums1[i]
        //for j in range(n):
        //    nums1[m+j] = nums2[j]
        //nums1 = nums1.sort()
        for (int i = 0; i < m; i++) {
            nums1[i] = nums1[i];
        }
        for (int j = 0; j < n; j++) {
            nums1[m+j] = nums2[j];
        }
        sort(nums1.begin(), nums1.end());
    }
};